modules/minimap2/1.0/config/default.yaml

```diff
@@ -0,0 +1,36 @@
+lcr-modules:
+    
+    minimap2:
+
+        # TODO: Update the list of available wildcards, if applicable
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_fastq: "__UPDATE__"
+            reference_build: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        options:
+            minimap2: '-ax map-ont -L --MD -Y -R "@RG\tID:{sample_id}\tLB:{sample_id}\tPL:ONT\tSM:{sample_id}"'
```

Comment:
Can you please give in-line documentation what these options do? Something similar to other documentation, like [this](https://github.com/LCR-BCCRC/lcr-modules/blob/master/modules/liftover/2.0/config/default.yaml) for example